test-app/tests/integration/components/textarea-field-test.gts

```diff
@@ -0,0 +1,181 @@
+import { module, test } from 'qunit';
+
+import TextareaField from '@crowdstrike/ember-toucan-core/components/form/textarea-field';
+import { setupRenderingTest } from 'test-app/tests/helpers';
+
+module('Integration | Component | TextareaField', function (hooks) {
+  setupRenderingTest(hooks);
+
+  test('it renders', async function (assert) {
+    await render(<template>
+      <TextareaField @label="Label" data-textarea />
+    </template>);
+
+    assert.dom('[data-label]').hasText('Label');
+
+    assert
+      .dom('[data-hint]')
+      .doesNotExist(
+        'Expected hint block not to be displayed as a hint was not provided'
+      );
+
+    assert.dom('[data-textarea]').hasTagName('textarea');
+    assert.dom('[data-textarea]').hasAttribute('id');
+    assert.dom('[data-textarea]').hasClass('text-titles-and-attributes');
+
+    assert
+      .dom('[data-error]')
+      .doesNotExist(
+        'Expected hint block not to be displayed as an error was not provided'
+      );
+  });
+
+  test('it renders with a hint', async function (assert) {
+    await render(<template>
+      <TextareaField @label="Label" @hint="Hint text" data-textarea />
+    </template>);
+
+    let hint = find('[data-hint]');
+
+    assert.dom(hint).hasText('Hint text');
+    assert.dom(hint).hasAttribute('id');
+
+    let hintId = hint?.getAttribute('id') || '';
+    assert.ok(hintId, 'Expected hint ID to be truthy');
+
+    let describedby =
+      find('[data-textarea]')?.getAttribute('aria-describedby') || '';
+    assert.ok(
+      describedby.includes(hintId),
+      'Expected hintId to be included in the aria-describedby'
+    );
+  });
+
+  test('it renders with an error', async function (assert) {
+    await render(<template>
+      <TextareaField @label="Label" @error="Error text" data-textarea />
+    </template>);
+
+    let error = find('[data-error]');
+
+    assert.dom(error).hasText('Error text');
+    assert.dom(error).hasAttribute('id');
+
+    let errorId = error?.getAttribute('id') || '';
+    assert.ok(errorId, 'Expected error ID to be truthy');
+
+    let describedby =
+      find('[data-textarea]')?.getAttribute('aria-describedby') || '';
+    assert.ok(
+      describedby.includes(errorId),
+      'Expected errorId to be included in the aria-describedby'
+    );
+
+    assert.dom('[data-textarea]').hasAttribute('aria-invalid', 'true');
+  });
+
+  test('it sets aria-describedby when both a hint and error are provided using the hint and error IDs', async function (assert) {
+    await render(<template>
+      <TextareaField
+        @label="Label"
+        @error="Error text"
+        @hint="Hint text"
+        data-textarea
+      />
+    </template>);
+
+    let errorId = find('[data-error]')?.getAttribute('id') || '';
+    assert.ok(errorId, 'Expected error ID to be truthy');
+
+    let hintId = find('[data-hint]')?.getAttribute('id') || '';
+    assert.ok(hintId, 'Expected hint ID to be truthy');
```

Comment:
`hint ID` -> `hintId`

I think either ALL the descriptions should be `error ID` or `errorId` I mean....(one format or another)

Reply:
Yeah that makes sense.  I more think of these as the hint's ID, rather than being exactly tied to the variable name of `hintId`, but I definitely see your point! Went ahead and updated!